src/transcribe_endpoint.py

```
@@ -0,0 +1,181 @@
+            decoded_data = base64.b64decode(audio_data)
+
+            # Check file size (max 25MB)
+            if len(decoded_data) > 25 * 1024 * 1024:
+                return False
+
+            # Check format against allowlist
+            allowed_audio_formats = {'wav', 'mp3', 'flac', 'ogg', 'm4a', 'aac'}
+            if audio_format.lower() not in allowed_audio_formats:
+                return False
+
+            return True
```

Ruff (0.13.0)

Consider moving this statement to an `else` block

(TRY300)